src/components/Button/Button.js

```diff
@@ -5,8 +5,17 @@ import icons from '../Icon/icons';
 import SubmitButton from './SubmitButton';
 import get from 'extensions/themeGet';
 
+/**
+ * NOTE: all paddings in button are adjusted to accomodate
+ * border widths on both sides. If border widths are changed,
+ * also change padding to compensate!
+ *
+ * Vertical paddings are also displaced to accomodate text cap and base lines.
+ * Basically, 1px is added to the top, 1px subtracted from the bottom.
```

Comment:
Doesn't `box-sizing` fix this issue?

Reply:
Nope, since the height is determined by padding, not an explicit `height` property.

Can't set `height` explicitly since the text inside might need to wrap in some circumstances.